### DISCUSSION TITLE
Reversed vehicle rating multiplier

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#19443] [Feature requests] [answered]
@ZeeMajora writes in discord:
> I think we need a "reverseRatingMultiplier" field that would basically work as a rating multiplier that'd only be applied if the trains are reversed

In my opinion this would be a good feature and should be a follow-up PR to https://github.com/OpenRCT2/OpenRCT2/pull/19305

If this is implemented, the RCT2 6-seater reversed trains can be consolidated into a single object, and the RCT1 reversed woodie trains can have the same.

**Comment by ZeeMaji** [comment]
The RCT1 reversed steel trains as well, this field should also probably have some non-zero default values that'd be used for all objects if they're reversed so that way every ride has it's ratings changed if the vehicles are reversed.
It'd also probably make sense to combine it with the vehicle's normal rating multipliers if the vehicle has any.

**Comment by ZeeMaji** [comment, marked as answer]
Implemented in #20091 but for ride types rather than ride vehicle objects.
